recipes/colorama/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "colorama" %}
+{% set version = "0.3.7" %}
+{% set build = 0 %}
+{% set md5 = "349d2b02618d3d39e5c6aede36fe3c1a" %}
```

Comment:
I prefer sha256, but this is a minor thing.  md5 is broken from a security standpoint.

Not a sticking point, just a comment - change if you want, or don't if you don't.